Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN mkdir -p /app/maiko
 WORKDIR /app
 COPY ./* ./
```

Comment:
Why copy the directory?  Wouldn't it be better to mount the medley directory when Docker starts?  Copying Medley into the Docker image bloats it to 2.9 GB.  Without the directory the Docker image is only 376 MB.  By keeping it separate you could have one Docker container that runs multiple instances of Medley,  One could easily imagine the Maiko engine being on a different release cadence then Medley.